packages/eas-cli/src/standaloneBuilds.ts

```diff
@@ -0,0 +1,43 @@
+import { Platform } from '@expo/config';
+
+import { apiClient } from './api';
+import { ensureLoggedInAsync } from './user/actions';
+
+interface StandaloneBuildParams {
+  platform: Platform;
+  id?: string;
+  slug: string;
+  owner?: string;
+}
+
+export type Build = any;
+
+// Legacy Turtle v1 builds
+export async function getStandaloneBuilds(
+  { platform, slug, owner, id }: StandaloneBuildParams,
+  limit?: number
+): Promise<Build[]> {
+  await ensureLoggedInAsync();
```

Comment:
the function that is essentially api call wrapper shouldn't be triggering login prompt